gptqmodel/models/base.py

```diff
@@ -104,6 +102,7 @@ def _prepare_dataset_for_quantization(
             self,
             calibration_dataset: List[Dict[str, Union[List[int], torch.LongTensor]]],
             batch_size: int = 1,
+            tokenizer = None,
```

Comment:
missing type hint.. for tokenizer. check HF tokenizer base cls type. and use `tokenizer: Optional[BaseTokenCls]